src/main/java/com/fryrank/controller/ReviewController.java

```diff
@@ -43,6 +44,11 @@ public GetAllReviewsOutput getAllReviews(
         }
     }
 
+    @GetMapping(value = REVIEWS_URI_TOP_10)
+    public GetAllReviewsOutput getReviewsTen() {
```

Comment:
Can we make this more modular by receiving an input for the number of reviews to receive instead of having 10 specifically?